packages/lesswrong/components/form-components/FormComponentDateTime.tsx

```diff
@@ -31,7 +35,9 @@ const FormComponentDateTime = ({ path, value, name, label, classes, position },
   const date = value ? (typeof value === 'string' ? new Date(value) : value) : null;
 
   return <FormControl>
-    <InputLabel className={classes.label}>{ label }</InputLabel>
+    <InputLabel className={classes.label}>
+      { label } <span className={classes.timezone}>({moment().tz(moment.tz.guess()).zoneAbbr()})</span>
```

Comment:
Should this be `moment(date)` to cover the corner case where the date they are inputting is across a daylight savings time boundary?